src/app/about-button/about-button.component.ts

```diff
@@ -0,0 +1,52 @@
+import {Component, DoCheck, OnInit} from '@angular/core';
+declare var $: any;
+
+@Component({
+  moduleId: module.id,
+  selector: "about-button",
+  templateUrl: "about-button.component.html",
+  styleUrls: ["about-button.component.css"],
+})
+
+export class AboutButtonComponent implements DoCheck, OnInit {
+
+  timesClicked: any;

```

Comment:
Is there a point in tracking the number of times clicked?
All we want is for this to be a toggle button, no?

Maybe use one of the standard methods to implement a toggle button with Angular 2?
https://www.google.com/?ion=1&espv=2#q=angular%202%20toggle%20button